app/react/kubernetes/queries/query-keys.ts

```diff
@@ -0,0 +1,8 @@
+/** Kubernetes environment base query keys */
+export const queryKeys = {
+  base: (environmentId: number) => [
+    'environments',
+    environmentId,
+    'kubernetes',
+  ],
```

Comment:
```suggestion
  ] as const,
```

this will let typescript type this array, instead of just `return (string|number)[]`

Reply:
updated